templates/ui-frameworks/svelte/link-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{#if (and bidireccional (and to_referenceable (ne from_referenceable.hash_type 'AgentPubKey')))}}{{pascal_case (plural from_referenceable.name)}}For{{pascal_case to_referenceable.name}}.svelte{{¡if}}.hbs

```diff
@@ -5,13 +5,17 @@ import { getClient } from "../../contexts";
 from_referenceable.name}}Detail from './{{pascal_case from_referenceable.name}}Detail.svelte';
```

Comment:
I assume this is broken template syntax? I see it existed before this PR

Reply:
Well spotted! https://github.com/holochain/scaffolding/pull/486/commits/e0c732ce4bc260be45b18eae6a58da10e43a9d76